libs/net/dal/Services/WorkOrderService.cs

```diff
@@ -209,7 +209,7 @@ select new
             query = query.Where(c => filter.Status.Contains(c.Status));
 
         if (filter.ContentId.HasValue)
-            query = query.Where(c => c.ContentId == filter.ContentId || EF.Functions.JsonContains(c.Configuration, $"{{\"contentId\":{filter.ContentId}}}"));
+            query = query.Where(c => c.ContentId == filter.ContentId);
```

Comment:
This was added at one point because of historical data not including the `ContentId` property.  Instead it referenced the content in the configuration JSON object.  I suspect removing it will only impact older data.